### ISSUE TITLE
Support uploading  AQATapFiles by URL

### ISSUE THREAD

**Issue by sophia-guo** [adoptium/github-release-scripts#155]
Currently upload  AQATapFiles is reusing UPSTREAM_JOB_NAME and UPSTREAM_JOB_NUMBER. AQATapFiles is one final tar.gz file. It is not necessary to be archived in jenkins jobs. it can be even located at https://github.com/adoptium/aqa-tests/files. 

This feature can increase the flexibility and be user friendly.

**Comment by smlambert** [comment]
Yes, another note about this is that we currently end up having to amalgamate extra Grinder TAP files into the final uploaded set of results because we do not get through a full release pipeline with "all green" runs.  

Example for March release, 1 single JDK version, saw us need to include 19 extra Grinder TAP files into the final results bundle (https://github.com/adoptium/aqa-tests/files/14856184/AQAvitTapFiles_March2024.zip), this did not include the automatic rerun TAP files which should ideally also be included, so that the set of results that is linked to from the Adoptium Marketplace is a complete set, showing that all test cases were run and passed.

Related:
https://github.com/adoptium/ci-jenkins-pipelines/issues/373
https://github.com/adoptium/aqa-tests/issues/5015
https://github.com/adoptium/aqa-tests/issues/3713
https://github.com/adoptium/marketplace-api.adoptium.net/issues/33
